config/locales/exceptions.en.yml

```diff
@@ -78,6 +78,12 @@ en:
         title: Validation error
         code: 109
         status: 422
+      not_a_safe_host_error:
```

Comment:
Can we say it's "is not a whitelisted host"  or something rather than  "is not a safe host".  A dev without context might easily hit something like `{"errors":[{"title":"Bad Request","detail":"\"127.0.0.1\" is not a safe host","code":"110","status":"400"}]}` and get needlessly confused.

Reply:
Do you think InvalidVirtualHost would be better?

Reply:
yeah, that's better. Maybe all that we need is a comment somewhere.  I can see someone being confused about why 127.0.0.1 or 0.0.0.0 are blocked, especially when they initially join the project.